sass/dimensions.scss

```diff
@@ -0,0 +1,5 @@
+$num-of-columns: 12;
```

Comment:
You could check in Composer if we ever use column width that are odd, and whether they are really necessary. Gathering data from a real system to inform future architecture decisions is a good way to test hypotheses of needs.